TOC.md

```diff
@@ -216,219 +229,179 @@
+      + [Control Flow Functions](/functions-and-operators/control-flow-functions.md)
+      + [String Functions](/functions-and-operators/string-functions.md)
+      + [Numeric Functions and Operators](/functions-and-operators/numeric-functions-and-operators.md)
+      + [Date and Time Functions](/functions-and-operators/date-and-time-functions.md)
+      + [Bit Functions and Operators](/functions-and-operators/bit-functions-and-operators.md)
+      + [Cast Functions and Operators](/functions-and-operators/cast-functions-and-operators.md)
+      + [Encryption and Compression Functions](/functions-and-operators/encryption-and-compression-functions.md)
+      + [Information Functions](/functions-and-operators/information-functions.md)
+      + [JSON Functions](/functions-and-operators/json-functions.md)
+      + [Aggregate (GROUP BY) Functions](/functions-and-operators/aggregate-group-by-functions.md)
+      + [Window Functions](/functions-and-operators/window-functions.md)
+      + [Miscellaneous Functions]](/functions-and-operators/miscellaneous-functions.md)
+      + [Precision Math](/functions-and-operators/precision-math.md)
+      + [List of Expressions for Pushdown](/functions-and-operators/expressions-pushed-down.md)
+    + [Constraints](/constraints.md)
+    + [Generated Columns](/generated-columns.md)
+    + [SQL Mode](/sql-mode.md)
+    + Transactions
+      + [Overview](/transaction-overview.md)
+      + [Isolation Levels](/transaction-isolation-levels.md)
+      + [Optimistic Transactions](/optimistic-transaction.md)
+      + [Pessimistic Transactions](/pessimistic-transaction.md)
+    + Garbage Collection (GC)
+      + [Overview](/garbage-collection-overview.md)
+      + [Configuration](/garbage-collection-configuration.md)
+    + [Views](/views.md)
+    + [Partitioning](/partitioned-table.md)
+    + [Character Set and Collation](/character-set-and-collation.md)
+    + System Tables
+      + [`mysql`](/system-tables/system-table-overview.md)
+      + [`information_schema`](/system-tables/system-table-information-schema.md)
+      + sql-diagnosis
+        + [`cluster_info`](/system-tables/system-table-cluster-info.md)
+        + [`cluster_hardware`](/system-tables/system-table-cluster-hardware.md)
+        + [`cluster_config`](/system-tables/system-table-cluster-config.md)
+        + [`cluster_load`](/system-tables/system-table-cluster-load.md)
+        + [`cluster_systeminfo`](/system-tables/system-table-cluster-systeminfo.md)
+        + [`cluster_log`](/system-tables/system-table-cluster-log.md)
+        + [`metrics_schema`](/system-tables/system-table-metrics-schema.md)
+        + [`metrics_tables`](/system-tables/system-table-metrics-tables.md)
+        + [`metrics_summary`](/system-tables/system-table-metrics-summary.md)
+        + [`inspection_result`](/system-tables/system-table-inspection-result.md)
+        + [`inspection_summary`](/system-tables/system-table-inspection-summary.md)
+  + UI
+    + TiDB Dashboard
```

Comment:
Please help update this section after all your dashboard PRs are merged. @TomShawn

Reply:
OK

Reply:
Updated